plugins/cros_ec-usb/fu-cros_ec-usb-common.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2020 Benson Leung <bleung@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <string.h>
+
+#include "fu-cros_ec-usb-common.h"
+
+gboolean
+fu_cros_ec_usb_parse_version (const gchar *version_raw,
+			      struct cros_ec_version *version,
+			      GError **error)
+{
+	g_auto(GStrv) v_split = NULL;
+	g_auto(GStrv) marker_split = NULL;
+	g_auto(GStrv) triplet_split = NULL;
+
+	if (NULL == version_raw || 0 == strlen (version_raw)) {
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_INTERNAL,
+				     "No version string to parse.");
```

Comment:
no fullstops at the end of the error message please.

Reply:
Done.